src/views/SignTransactionLedger.vue

```diff
@@ -1,8 +1,11 @@
 <template>
-    <div class="container">
+    <div v-if="request.kind !== 'cashlink' || !!cashlink" class="container">
```

Comment:
I don't understand this condition? Always show when the request is NOT of type 'cashlink' or when there is a cashlink variable? How does this make sense? Especially since you still check for the 'cashlink' request type in the conditions below.

Reply:
If the `request.kind` is `'cashlink'` the template should only be rendered, if `cashlink` is defined to avoid accessing the properties on the cashlink if it's undefined.
I can reformulate this to `request.kind === 'cashlink' ? !!cashlink : true".

Reply:
Don't know if that's better :D I did not find the condition formulation confusing, just that it's there in the first place :stuck_out_tongue: